src/instructlab/model/download.py

```diff
@@ -18,7 +17,7 @@
 @click.command()
 @click.option(
     "--repository",
-    default="instructlab/merlinite-7b-lab-GGUF",  # TODO: add to config.yaml
+    default=config.DEFAULTS.MERLINITE_GGUF_REPO,  # TODO: add to config.yaml
```

Comment:
elsewhere you import `from instructlab.configuration import DEFAULTS` which I kinda like better.
non-blocking, just a stylistic comment.

Reply:
Okay we'll use this instead.

Reply:
Resolved this.